.circleci/check_version.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Can you add a bit more of an explanation of what this commit achieves to the commit message - i.e. what is the behaviour change?

Reply:
Done